static/assets/js/phenopolis.js

```diff
@@ -168,7 +168,7 @@ if (!PP) {
                 $('#change_password_successful').show();
                 $("#change_password_successful").text(data.success);
             },
-            error: function (xhr, msg) {
+            error: function (data, msg) {
```

Comment:
I tried a few different things with xhr but couldn't get it to display messages such as "You do not have permission to change the password for username 'demo'.". What do you suggest?